### PR TITLE
fix #50: return type should support binary string

### LINKED ISSUE #50: PDK fails to unpack binary string returned by `request.get_raw_body()`
See https://github.com/Kong/kong-js-pdk/issues/156

### PR DESCRIPTION
**Pull request by Suika-Kong**
fix #50

### CODE REVIEW

**Comment by Suika-Kong** [comment]
The lint problem is not introduced by this PR.

**Comment by Suika-Kong** [comment]
Note this is a breaking change involving the interface. We still need further discussion.

File: kong_pdk/pdk/kong/response.py [hunk 1]
```diff
@@ -337,9 +337,10 @@ def get_headers(max_headers: Optional[number]) -> Tuple[table, str]:
         :rtype: str
         """
         pass
-
+    
+    # this function's return type is modified mannually as body can be arbitrary binary string
     @staticmethod
-    def get_raw_body() -> str:
+    def get_raw_body() -> Union[str, bytes]:
```
Comment (Suika-Kong) [inline kong_pdk/pdk/kong/response.py:343]:
```suggestion
    def get_raw_body() -> bytes:
```

File: kong_pdk/pdk/kong/service/response.py [hunk 2]
```diff
@@ -135,8 +135,9 @@ def get_headers(max_headers: Optional[number]) -> Tuple[table, str]:
         """
         pass
 
+    # this function's return type is modified mannually as body can be arbitrary binary string
     @staticmethod
-    def get_raw_body() -> str:
+    def get_raw_body() -> Union[str, bytes]:
```
Comment (Suika-Kong) [inline kong_pdk/pdk/kong/service/response.py:140]:
```suggestion
    def get_raw_body() -> bytes:
```

**Review by Suika-Kong — COMMENTED** [review]
new method to accept both binary data and text string

**Review by fffonion — APPROVED** [review]

**Review by fffonion — APPROVED** [review]

**Pull request merged**